### ISSUE TITLE
add GH action to check/publish animint book

### ISSUE THREAD

**Issue by tdhock** [animint/animint2#76]
these days before merging a PR, I am re-building https://github.com/tdhock/animint-book to make sure nothing breaks.
this suggests we should add a GH action which does this automatically for every push.
For publishing it would be nice to have at least the master branch.

**Comment by Faye-yufan** [comment]
Hi @tdhock , do you think checking for compilation errors in the GH action is enough? I recall when I worked on the `color_off` function, most issues from the manual seemed to be related to visual effects, which required a visual inspection. Do you feel there's a need to create tests specifically for these visual effects?

**Comment by tdhock** [comment]
you are right it probably will require a manual visual inspection, but it would be useful to have that automatically built and deployed via actions (rather than on my own laptop). hopefully things like color_off should be covered in tests?

**Comment by tdhock** [comment]
this is happening in https://github.com/animint/animint-manual-en/blob/main/.github/workflows/build-book.yml
